### PR TITLE
DOT ECS Migration doc

### PR DESCRIPTION
**Pull request by JAMSUPREME**
### JIRA Ticket

[SDC-2494](https://securedatacommons.atlassian.net/browse/SDC-2494)

### Summary

Documentation for how we'll manage two clouds simultaneously

### CODE REVIEW

File: ECS_MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+```
+resource "aws_cognito_identity_provider" "SamlDotActiveDirectoryIdp" {
+  user_pool_id  = var.user_pool_id
+  provider_name = "${var.environment}-dot-ad"
+  provider_type = "SAML"
+}
+
+# New pool that is not used in our current account, but mirrors the settings of var.user_pool_id pool
+# For testing, we can update the user_pool_id to be the ID of this resource, but DO NOT use a ref above
+resource "aws_cognito_user_pool" "new_pool" {
```
Comment (aaronFranssell) [inline ECS_MIGRATION.md:76]: What do you think about these existing resources getting imported into Terraform in our public account? If we used this strategy we wouldn't need a long lived ECS branch and risk divergence between master and `ecs-migration`.

Perhaps I'm missing your point...

Comment (JAMSUPREME) [reply ECS_MIGRATION.md:76]: Just being risk averse. We could try that, but I'm a bit worried we'd botch one or more settings. Perhaps create it as a one-off so we can compare with the existing one, and then if they do appear identical, try an import?

Comment (aaronFranssell) [reply ECS_MIGRATION.md:76]: Yes that works. Lets try it out for the initial story you are picking up.

File: ECS_MIGRATION.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+# Webportal-specific guidance
+
+Webportal deploys are already composed of two parts:
+
+- `prod_deploy.sh` - Pushes the angular files into a target S3 bucket for static asset hosting
+- `python3 deploy.py --environment production` - Uses Chalice to push our API Gateways and lambdas into AWS
+
+I am proposing we add a third step:
+- `terraform apply` which will deploy any non-chalice resources into AWS.
```
Comment (TiannaW24) [inline ECS_MIGRATION.md:13]: Small thing but can we put the full init and apply here? Not 100% if this document is meant to be long-lasting, but if it is, I'd want both commands here.

Comment (JAMSUPREME) [reply ECS_MIGRATION.md:13]: Yeah - I'll add it once I actually put one in. This was more for concepts before I start coding.

**Pull request merged**